### PR TITLE
Add 2 new notifications modes (message, flash)

### PR DESCRIPTION
**Pull request by deathbeam**
- Change the notification mode selection to be dropdown
- Add chat message notification mode that will send game message on
notificiation
- Add screen flash notification that will flash screen for few ticks

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

Preview:
peek 2018-04-30 15-01
screenie

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java [hunk 1]
```diff
@@ -80,32 +81,21 @@ default boolean gameAlwaysOnTop()
...
 	@ConfigItem(
-		keyName = "notificationTray",
-		name = "Enable tray notifications",
-		description = "Enables tray notifications",
+		keyName = "notificationMode",
+		name = "Notification mode",
+		description = "Determines mode of notifications",
 		position = 5
 	)
-	default boolean enableTrayNotifications()
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java]: Is it no longer possible to disable notifications with this?

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java]: Oh, there should be OFF mode, oops.

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java]: Done

**Comment by deathbeam** [comment]
So as this is approved, can it be also merged?

**Pull request merged**